builtin/plugins/coin/coin.go

```diff
@@ -119,6 +119,22 @@ func (c *Coin) MintToGateway(ctx contract.Context, req *MintToGatewayRequest) er
 	return mint(ctx, gatewayAddr, &req.Amount.Value)
 }
 
+
+// MintToDPOS adds loom coins to the loom coin DPOS contract balance, and updates the total supply.
+func (c *Coin) MintToDPOS(ctx contract.Context, req *MintToDPOSRequest) error {
```

Comment:
Don't need this, the DPOS contract will only mint ERC20

Reply:
Done.